src/lib.rs

```diff
@@ -832,14 +842,14 @@ impl<T: fmt::Debug> fmt::Debug for Grid<T> {
                         write!(
                             f,
                             " {item:width$.precision$?}",
-                            width = width,
-                            precision = precision
+                            // width = width,
```

Comment:
Why did you commented out this?

Reply:
It isn't necessary - format can pull in identifiers from the surrounding scope.

It's a relatively new feature - if you want to keep an older MSRV, just uncomment it.

Reply:
Ah cool 😎 did not know that. Fine for me if it works